src/main/resources/schema.sql

```
@@ -0,0 +1,6 @@
+CREATE TABLE product (
+                         id BIGINT AUTO_INCREMENT PRIMARY KEY,
+                         name VARCHAR(255) NOT NULL,
+                         price INT NOT NULL,
+                         image_url VARCHAR(500) NOT NULL
```

VARCHAR에 문자열 제한이 있으니 데이터베이스에 저장하기 전에 검증 로직에도 들어가면 좋을 것 같아요~
개발엔 빠른 실패(Fail Fast)라는 개념이 있어요. 문제가 생기면 즉시 알려줘야한다는 개념이고 그렇지 않고 진행하다 작업의 중간에서 실패하면 롤백하기도 어렵고 디버깅하기도 어려워서 생긴 개념이예요.
이런 관점에서 비추어볼 때, 데이터베이스에 저장하는 과정에서 실패하기보단 요청을 받았을 때 실패하는 것이 훨씬 추적하기도 쉬워요. 특히 현업처럼 요청을 받고 데이터베이스에 저장하는 과정이 긴 경우에는 더더욱 필요한 개념입니다!